src/main/java/seedu/workbook/logic/commands/AddCommand.java

```diff
@@ -65,7 +72,12 @@ public CommandResult execute(Model model) throws CommandException {
 
         model.addInternship(toAdd);
         model.commitWorkBook();
-        return new CommandResult(String.format(MESSAGE_SUCCESS, toAdd));
+
+        if (model.internshipStageHasNoTips(toAdd)) {
```

Comment:
I don't think it makes sense for the `#internshipStageHasNoTips` to be defined in `Model` and `ModelManager`. We have access to an Internship object via `toAdd` anyway, why not define these methods within the `Internship` class?